apps/api/src/services/programs.ts

```diff
@@ -34,27 +37,30 @@ export class ProgramsService {
         division: degree.division,
       })
       .from(majorSpecialization)
+      .where(query.id ? eq(majorSpecialization.id, query.id) : sql`true`)
```

Comment:
```sql'true'``` is a valid solution here, but we'll want to use `undefined` since it better denotes that we don't  want to execute a where clause (rather than executing WHERE TRUE)

```suggestion
      .where(query.id ? eq(majorSpecialization.id, query.id) : undefined)
```

Reply:
Tried to use `undefined` in [8905d30](https://github.com/icssc/anteater-api/pull/97/commits/8905d30290dfad72a96c33d81739ac2404aa37bd) which resulted in a staging failure which was resolved with [90749e6](https://github.com/icssc/anteater-api/pull/97/commits/90749e698c70dad0533bef934b2428ffae85bfe1)

Reply:
Hmmm that's odd, let me do some digging

Reply:
> Tried to use `undefined` in [8905d30](https://github.com/icssc/anteater-api/pull/97/commits/8905d30290dfad72a96c33d81739ac2404aa37bd) which resulted in a staging failure. Resolved with [90749e6](https://github.com/icssc/anteater-api/pull/97/commits/90749e698c70dad0533bef934b2428ffae85bfe1).

@waterkimchi looks like it may have been a one time issue with the github action, [90b34d2](https://github.com/icssc/anteater-api/pull/97/commits/90b34d2d53a2e46ed72bc31156f7348a5cfbce74) staged without error. If you come across this again, feel free to lmk to rerun the action.